pkg/grpc/controller/controller.go

```diff
@@ -0,0 +1,51 @@
+package grpccontroller
+
+import (
+	"context"
+	"fmt"
+
+	pb "monorepa/pkg/grpc/proto"
+
+	"github.com/google/uuid"
+)
+
+type GRPCСontroller struct {
+	client pb.GrpcServiceClient
+}
+
+func New(cli pb.GrpcServiceClient) GRPCСontroller {
+	return GRPCСontroller{
+		client: cli,
+	}
+}
+
+type Item struct {
```

Comment:
May be better to separate as model(general or other file)

Reply:
I will make a unified model when all my pull requests are merged.